text/docs-code-contributions.md

```diff
@@ -0,0 +1,59 @@
+- Start Date: 2019-02-07)
+- RFC PR: 
+- Gatsby Issue:
+
+# Summary
+
+Reorganize the contributor documentation to aid in Gatsby core and docs development.
+
+# Motivation
+
+The Gatsby.js docs, while wonderful, are light on coding instructions for contributors. With dedicated sections in the Contributing docs, it will be easier to learn how to clone the repository, where to find certain things like docs or plugins, how to run tests, and insider infrastructure knowledge required to submit a successful pull request on the first try.
+
+A lot of this information already lives on the "How to Contribute" page, but it's buried. It would help discoverability if this information lived on separate pages.
+
+# Detailed design
+
+I propose that we add some pages to the existing contributor documentation and move some content around. Here is an IA diagram that (briefly) reflects the current structure and what I think we should add (open for discussion, of course):
+
+Contributing
+- Community
+    - Pair Programming Sessions
+    - Where to participate in the community (this was in RFCs, perhaps it could be reintroduced and content moved from "How to contribute"?)
+    - How to run a Gatsby workshop*
+    - How to pitch Gatsby*
+- Code of Conduct
+- Gatsby Style Guide
+- How to contribute
+    - How to file an issue
+    - How to label an issue
+    - Free swag for contributors
+    - Contributing to the documentation
+        - Docs templates
+    - Contributing to the repo
```

Comment:
Maybe it'd be good to have a dedicated 'Setting up your local development environment' page that covers the first bit of the [_Contributing to the repo_ setup section](https://www.gatsbyjs.org/docs/how-to-contribute/#contributing-to-the-repo) (cloning the repo, installing yarn, running the bootstrap step, and using gatsby-dev-cli). 

It could be referenced from the _Contributing to the repo_, and _Settting up test environments_ pages, and possibly the _Contributing to the documentation_ page?

Reply:
Oh I guess this is covered by your later comment:

> There may be additional subpages added for more detailed technical instructions.

👍 from me!